packages/nvidia_nat_vanna/src/nat/plugins/vanna/execute_db_query.py

```
@@ -0,0 +1,237 @@
+                dataframe_info=dataframe_info,
+            )
+
+            if original_row_count > config.max_rows:
+                response.limited_to = config.max_rows
+                response.truncated = True
+
+            # Yield final result as ExecuteDBQueryOutput
+            yield response
+            # Note: Engine is left alive; connections are managed internally by SQLAlchemy pool
+
+        except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)